images/manageiq-orchestrator/container-assets/entrypoint

```diff
@@ -43,6 +43,13 @@ EOS
   bin/rails runner "$script"
 }
 
+function create_initial_group() {
+  if [ "$ADMIN_GROUP" != "" ]; then
+    echo "== Creating initial group "$ADMIN_GROUP" =="
+    bin/rails runner "MiqGroup.create!(:miq_user_role => MiqUserRole.find_by(:name => 'EvmRole-super_administrator'), :description => ENV['ADMIN_GROUP'], :tenant => Tenant.root_tenant)"
+  fi
+}
```

Comment:
is DB always empty/just seeded when this happens ?

Reply:
Oh, right this isn't idempotent. Let me fix.

Reply:
LOL you just wanted to say "idempotent", didn't you?

Reply:
Maybe .... :grin: